README.md

```diff
@@ -1,6 +1,23 @@
 # Welcome to Crowbar Angular Prototype
 [![Build Status](https://travis-ci.org/crowbar/crowbar-ui.svg?branch=master)](https://travis-ci.org/crowbar/crowbar-ui)
 
+### Setting up a development environment with Docker
+
+1. make sure `docker` is installed and the `docker` daemon is running.
```

Comment:
From my point of view, a development setup guide should guide the user to set up the environment as fast and easily as possible.
Two or three steps detailing how to install docker and setup the daemon (if required) would help any user of this guide to get the thing up and running